TopAnalysis/python/postprocessing/fcncTriLepton.py

```diff
@@ -63,7 +64,7 @@ def initReaders(self,tree):
         objName = "Electron"
         setattr(self, "b_n%s" % objName, tree.valueReader("n%s" % objName))
         for varName in ["pt", "eta", "phi", "mass", "charge",
-                        "pfRelIso03_all", self.eleIdName, "deltaEtaSC", "eCorr",]:
+                        "pfRelIso03_all", self.eleIdName, "deltaEtaSC", "eCorr", "vidNestedWPBitmapSum16",]:
```

Comment:
Should be able to change this vid bitmap name - to run on 2017+ datasets for the future.